src/main/java/frc/robot/subsystems/ArmSystem.java

```diff
@@ -23,17 +32,33 @@ public ArmSystem() {
 
         motor.restoreFactoryDefaults();
 
+        setUpShuffleboard();
+    }
+
+    private void setUpShuffleboard() {
+        ShuffleboardTab tab = ShuffleboardUtils.getArmIntakeShooterTab();
+
+        angleEntry = tab.add("Arm Angle", 0.0)
+                .withWidget(BuiltInWidgets.kGyro)
+                .withProperties(Map.of("Counter clockwise", true))
+                .withPosition(0, 1)
+                .withSize(4, 4)
+                .getEntry();
+
+        ShuffleboardLayout subsystemsLayout = ShuffleboardUtils.getArmIntakeShooterSubsystemsLayout();
+        subsystemsLayout.add("Arm", this)
+                .withPosition(0, 0);
     }
 
     public boolean reachedATargetAngle(double targetAngle) {
         return MathUtil.isNear(targetAngle, getAbsEncoderPositionDegrees(), NEAR_ANGLE_TOLERANCE);
     }
 
-    public void moveToAngle(double output) {
-        if ((output < 0 && getAbsEncoderPositionDegrees() > RobotMap.ARM_MAX_ANGLE) || output > 0 && getAbsEncoderPositionDegrees() < RobotMap.ARM_MIN_ANGLE){
+    public void move(double power) {
```

Comment:
again, don't make changes not relevant to your branch. Especially naming changes.

We are working in a team, you do not have free rein to change things at will

Reply:
Most of this code was written before I heard your rant on the importance of legacy code. I will reverse these changes.